mono/tests/test-driver

```diff
@@ -29,9 +29,10 @@ if ($test =~ /.*\|.*/) {
 	#This is a silly workaround, but all tests that use extra parameters need a larger timeout.
 	$timeout_in_minutes = 5;
 }
+$ENV{'TEST_DRIVER_TIMEOUT_SEC'} = $timeout_in_minutes * 60;
 
 $| = 0;
-print "Testing $test... ";
+print "Testing $test (TEST_DRIVER_TIMEOUT_SEC=$ENV{'TEST_DRIVER_TIMEOUT_SEC'})... ";
```

Comment:
I'm not sure we really need to output the env var here, it'll just spam the output.